src/actions/actions.test.js

```diff
@@ -200,7 +200,7 @@ describe('actions', () => {
     );
   });
 
-  it('should create an action to toggle an array of tags on/off', () => {
+  it('should create an action to toggle an array of modular pipelines on/off', () => {
```

Comment:
This change is irrelevant to the scope of my ticket but getting this change in to fix a wording issue I spot while working on this PR